_rules/95e043.md

```diff
@@ -0,0 +1,202 @@
+		<!-- Video Controls -->
+		<div id="video-controls">
+			<button type="button" id="play-pause" class="play">Play</button>
+			<button type="button" id="mute">Mute</button>
+		</div>
+	</div>
+	<script src="no-autoplay.js"></script>
+</body>
+```
+
+### Failed example 4
+
+The `video` element autoplays and has a control mechanism but the control buttons do not have an accessible name.
+
+``` html
+<head>
+<style>
+ button {color: #000;}
+ button:hover {cursor: pointer;	cursor: pointer; background-color: grey;  color: white;}
+</style>
+</head>
+<body>
+	<div id="video-container">
+		<!-- Video -->
+		<video id="video" autoplay="true">
+		  <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4">
+	   	  <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+		</video>
+		<!-- Video Controls -->
+		<div id="video-controls">
+      		<button type="button" id="play-pause" class="play"></button>
+		<button type="button" id="mute"></button>
+		</div>
+	</div>
+	<script src="https://github.com/auto-wcag/auto-wcag/tree/Brynanders-patch-3/assets/js/no-autoplay.js"></script>
+</body>
+```
+
+### Failed example 5
+
+The `video` element autoplays and has a control mechanism but the mechanism is not one of the first 5 focusable elements in the document.
+
+``` html
+<head>
+<style>
+button {color: #000;}
+button:hover {cursor: pointer;	cursor: pointer; background-color: grey;  color: white;}
+.play {background-color: lightgreen;}
+</style>
+</head>
+<body>
+	<div id="video-container">
+		<!-- Video -->
+		<video id="video" autoplay="true">
+		<source src="../test-assets/rabbit-video/video.mp4" type="video/mp4">
+	    	<source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+		  </p>
+		</video>
+		<!-- Video Controls -->
+		<div id="video-controls">
+				<button type="button">Button 1</button>
+    		<button type="button">Button 2</button>
+    		<button type="button">Button 3</button>
+    		<button type="button">Button 4</button>
+    		<button type="button">Button 5</button>
+     	 	<button type="button" id="play-pause" class="play"></button>
+		</div>
+	</div>
+	<script src="https://github.com/auto-wcag/auto-wcag/tree/Brynanders-patch-3/assets/js/no-autoplay.js"></script>
+</body>
+```
+
+### Inapplicable
+
+### Inapplicable example 1
+
+The `video` element is `muted`.
+
+``` html
+ <video autoplay="true" muted="true">
+  <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+### Inapplicable example 2
+
+The `video` element `src` file has no audio output.
+
+``` html
+ <video autoplay="true">
+  <source src="../test-assets/rabbit-video/video-with-incorrect-voiceover.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video-with-incorrect-voiceover.webm" type="video/webm" />
+</video>
+```
+
+### Inapplicable example 3
+
+The `audio` element does not `autoplay` attribute.
```

Comment:
"does not autoplay" or "does not have an `autoplay` attribute with a value of `true`"